src/utils/init.ts

```diff
@@ -35,13 +35,11 @@ type InitOptions = {
   userId?: string;
   preloadForms?: string[];
   userTracking?: 'cookie' | 'fingerprint';
-  authId?: string;
-  authEmail?: string;
-  authPhoneNumber?: string;
   language?: string;
 } & DeprecatedOptions;
 
 type InitState = {
+  authId?: string;
```

Comment:
I moved authId out of the init() fn call and into the auth gate like we talked about. but otherwise left the state itself here

since doing that I've made the authState in the auth gate, so lmk if you think it should stay or be moved there

Reply:
yeah IMO authId and authClient should probably both be moved there